components/TextFields/src/Theming/MDCTextInputControllerOutlined+MaterialTheming.m

```diff
@@ -28,7 +33,32 @@ - (void)applyThemeWithScheme:(nonnull id<MDCContainerScheming>)scheme {
 }
 
 - (void)applyColorThemeWithScheme:(nonnull id<MDCColorScheming>)colorScheme {
-  [MDCOutlinedTextFieldColorThemer applySemanticColorScheme:colorScheme toTextInputController:self];
+  UIColor *onSurfaceOpacity =
```

Comment:
Please update the Theming extension in a separate PR and then delete the themer in its own PR.

Reply:
I was definitely planning on deleting the themer in its own PR, after yet another PR for deprecating it. I can break out the theming extension change.

Reply:
This PR changes the theming extension: https://github.com/material-components/material-components-ios/pull/9223